apps/developer-portal/src/api/remote-repo-config.ts

```diff
@@ -0,0 +1,72 @@
+import {  InterfaceRemoteRepoListResponse, InterfaceRemoteRepoConfigDetails } from "../models";
+import { store } from "../store";
+import { HttpMethods } from "@wso2is/core/dist/src/models";
+import { AxiosHttpClient } from "@wso2is/http";
```

Comment:
We are using the new `@wso2is/oauth-web-worker` library to make API calls. See and example [here](https://github.com/wso2/identity-apps/blob/master/apps/developer-portal/src/api/application.ts#L19)

Reply:
Fixed with commit 8c08fbd